### PR TITLE
Improved cli installer

### PR DESCRIPTION
**Pull request by ghermans**
## Description
Improved the cli installer with [Laravel Prompts](https://laravel.com/docs/10.x/prompts#introduction)

## How To Test This?

run **php artisan bagisto:install**

image

### CODE REVIEW

File: packages/Webkul/Installer/src/Console/Commands/Installer.php [hunk 1]
```diff
@@ -235,16 +246,25 @@ protected static function getEnvAtRuntime(string $key): string|bool
...
+        );
+
+        if (!filter_var($adminEmail, FILTER_VALIDATE_EMAIL)) {
             $this->error('The email address you entered is not valid please try again.');
 
             return $this->createAdminCredential();
         }
 
-        $adminPassword = $this->ask('Please enter the <bg=green>Password</> for admin login:', 'admin123');
+        $adminPassword = Str::password();
```
Comment (devansh-webkul) [inline packages/Webkul/Installer/src/Console/Commands/Installer.php:267]: Password will always be 'admin123'.

**Comment by devansh-webkul** [comment]
@ghermans,

Please run `vendor/bin/pint` to fix the lint test.

**Comment by devansh-webkul** [comment]
@ashishkumar-webkul,

Please test the installer.

File: packages/Webkul/Installer/src/Console/Commands/Installer.php [hunk 2]
```diff
@@ -51,11 +58,11 @@ public function handle()
...
-        $this->info($this->call('storage:link'));
+        $this->call('storage:link');
 
         // Optimizing stuffs
         $this->warn('Step: Optimizing...');
-        $this->info($this->call('optimize'));
+        $this->call('optimize');
 
         $this->warn('Step: Composer autoload...');
         $result = shell_exec('composer dump-autoload');
```
Comment (devansh-webkul) [inline packages/Webkul/Installer/src/Console/Commands/Installer.php:68]: I don't think we need this. We can remove this.

File: packages/Webkul/Installer/src/Console/Commands/Installer.php (hunk 1 above)
Comment (ghermans) [reply packages/Webkul/Installer/src/Console/Commands/Installer.php]: I changed it back as requested to text and with a default value admin123, however i don't think that this is a secure way to setup the application.

**Comment by devansh-webkul** [comment]
Testcases are failing.

**Comment by ghermans** [comment]
> Testcases are failing.

Sorry i don't have more time to investigate why the tests are failing.

**Comment by devansh-webkul** [comment]
@amit-webkul,

Please check this PR and add the remaining code to your pull request. And also 155 test cases are failing also check that.

**Pull request closed without merging**